src/frontend/apps/standalone_site/src/features/Contents/features/featureLoader.tsx

```diff
@@ -17,20 +19,62 @@ import {
 import { liveContents, LiveRouter, routesLive } from './Live';
 import { videoContents, VideoRouter, routesVideo } from './Video';
 
-const featureLoader = () => {
+enum RESOURCES_CHOICES {
+  VIDEO = 'video',
+  WEBINAR = 'webinar',
+  CLASSROOM = 'classroom',
+}
+
+const featureLoader = (inactive_resources: string[]) => {
+  const featureRouter = [];
+  let featureRoutes = {};
+  const featureSamples: ((playlistId?: string) => FeatureSample)[] = [];
+  const featureShuffles = [];
+  if (!inactive_resources.includes(RESOURCES_CHOICES.VIDEO)) {
```

Comment:
```suggestion
const activeVideos = !inactive_resources.includes(RESOURCES_CHOICES.VIDEO);
  if (activeVideos) {
```
Just for improving readability a bit…

Reply:
https://github.com/openfun/marsha/pull/2276/commits/8ed8608006280d7e95893624ff943df89f8bd3ff#diff-7d8a5935e67b49566327f8eef4c2b80795e9f6baefec763b19f49a0cbc5ffa86R34